static/app/views/performance/transactionSummary/transactionOverview/transactionPercentage.tsx

```diff
@@ -31,8 +31,15 @@ export function TransactionPercentage({
   function getValueFromTotals(field, totalValues, unfilteredTotalValues) {
     if (totalValues) {
       if (unfilteredTotalValues) {
+        // Need to handle 0 case to avoid diving by 0
+        const volumeRatio =
+          unfilteredTotalValues[field] > 0
+            ? totalValues[field] / unfilteredTotalValues[field]
+            : 0;
+        const formattedPercentage =
+          volumeRatio < 0.0001 ? '<0.01%' : formatPercentage(volumeRatio);
```

Comment:
If the `count` numbers are a little volatile (eventual consistency, sampling, etc.) it might be good to use Lodash and do `volumeRatio = clamp(totalValues[field] / unfilteredTotalValues[field], 0, 1)` which will neatly cover the division by 0 case, and the "exceeds expectations" case where 100.001% shows up in the UI.

Reply:
Also, might be worth bringing up this `volumeRatio < 0.0001 ? '<0.01%'` case with Core UI (or someone else?) because that seems like a very useful formatting concern, so it might want to be part of `formatPercentage`!

Reply:
clamp wouldn't handle a case of division by 0 because it just checks that the resulting value is between min and max from what I see. the 0 denominator case still needs to be checked explicitly

Reply:
`_.clamp(100/0, 0, 1);` return `1` which I guess is wrong? Is seeing 100 of 0 events 100% of them, or 0% of them? 😅

Reply:
for all the cases i've seen it's always `0/0` in which case it should be 0% 🤔

Reply:
Makes sense to me!